server/src/main/java/org/elasticsearch/index/mapper/ConstantFieldType.java

```diff
@@ -35,11 +35,6 @@ public ConstantFieldType(String name, Map<String, String> meta) {
         assert isSearchable();
     }
 
-    @Override
-    public final boolean isAggregatable() {
```

Comment:
Are we changing it so Constant Fields are no longer aggregatable, or did you discover that the default implementation works for this field type?  The former seems like a breaking change.

Reply:
It turns out that `_tier` meta field doesn't support aggregating. It doesn't overwrite `fielddataBuilder(...)` method.
I removed this method because this assumes that all constant fields are aggregatable and this isn't the case. Only constant keyword and the index meta fields are aggregatable. 

Removing this method fixes this, the base class delegates to `hasDocValues()` method do determine whether a field is aggreagatable. Both constant keyword and index meta field types are mapped as if they have doc values, but the tier meta field is mapped that it doesn't have doc values. So the `isAggregatable()` in the super class is doing the right thing here.